sdk/servicebus/service-bus/src/queueClient.ts

```diff
@@ -60,7 +56,12 @@ export class QueueClient implements Client {
     throwErrorIfConnectionClosed(context);
     this.entityPath = String(queueName);
     this.id = `${this.entityPath}/${generate_uuid()}`;
-    this._context = ClientEntityContext.create(this.entityPath, ClientType.QueueClient, context);
+    this._context = ClientEntityContext.create(
+      this.entityPath,
+      ClientType.QueueClient,
```

Comment:
I see strange spacing here around the full stop.

Reply:
what do you see... I don't see anything

Reply:
I don't see it anymore! I think it was a github bug? It looked like `ClientType . QueueClient`.